.github/workflows/php-cs-fixer.yaml

```diff
@@ -1,19 +1,26 @@
-name: PHP Style
+name: "PHP-CS-Fixer"
 
-on: [push]
+on:
+    push:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+
+permissions:
+  contents: read
 
 jobs:
     php-cs-fixer:
+        permissions:
+          contents: write  # for stefanzweifel/git-auto-commit-action to push code in repo
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v2
               with:
                   ref: ${{ github.head_ref }}
 
             - name: PHP-CS-Fixer
-              uses: docker://oskarstark/php-cs-fixer-ga:2.19.0
-              with:
-                args: --config=.php_cs.dist --allow-risky yes
+              uses: docker://oskarstark/php-cs-fixer-ga:latest
```

Comment:
should we really go for the latest, or tag at least for a major version?

Reply:
See https://github.com/pimcore/pimcore/blob/11.x/.github/workflows/php-cs-fixer.yaml and other repros

Reply:
But yes I think we should set the latest major version. I will change it.

Reply:
@fashxp Done https://github.com/pimcore/data-hub/pull/854/commits/759bb046266ce4c2f399156cffe86d4e5fef0e11